properties/app.py

```diff
@@ -2,53 +2,65 @@
 
 from flask import Flask
 from flask import jsonify
-from flask import request
+# from flask import request
 from flask_cors import CORS
 
+
 app = Flask(__name__)
+app.url_map.strict_slashes = False
```

Comment:
Allows `/owners/` as well as `/owners` as candidates keep tripping up over this.

Reply:
Very good, it's annoying.